<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Actualizar_Tarifa</class>
 <widget class="QDialog" name="Actualizar_Tarifa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comTarifa">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #9fd5d1;
selection-color: rgb(72, 72, 72);
selection-background-color: transparent;
gridline-color: rgb(0, 0, 0);
font: 75 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tarifa mensual:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tarifa por hora:</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Actualizar_tarifa">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>Actualizar tarifa</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nueva_tarifa">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>Nueva tarifa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ActualTarifa">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Tarifa actual:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt 75 &quot;Times New Roman&quot;;
hover!pressed:border: 5px solid green;
background-color: #9BC499;</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancelar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt 75 &quot;Times New Roman&quot;;
hover!pressed:border: 5px solid green;
background-color: #DB7B65;



</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="NuevaTarifa">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #9fd5d1;
selection-color: rgb(72, 72, 72);
selection-background-color: transparent;
gridline-color: rgb(0, 0, 0);
font: 75 12pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>270</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>270</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagenes.qrc">:/Imagenes/Imagenes/Icon_2_0.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9fd5d1;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
